Resources/views/page_head_style.html.twig

```diff
@@ -5,6 +5,8 @@
     <link rel="stylesheet" type="text/css" href="{{ asset_url }}"/>
 {% endstylesheets %}
 
+<link rel="stylesheet" type="text/css" href="/bundles/ezplatformuiassets/vendors/flag-icon-css/css/flag-icon.min.css" />
```

Comment:
**@eZPlatformUIAssetsBundle** syntax causes path mismatch and flags cannot be displayed properly, [see link](http://symfony.com/doc/current/cookbook/assetic/asset_management.html#including-css-stylesheets).
Besides that, **bootstrap** isn't able to reach its font for _glyphicons_ (`../fonts/glyphicons-halflings-regular.woff2` - `404`) And afaik never was.
I think there are two possible ways to fix it:
1. Leave unused Bootstrap Glyphicons feature as unused and set cssrewrite filter. Also change syntax to:
```php
{% stylesheets filter="cssrewrite"
    'bundles/ezdemo/css/video.css'
    'bundles/ezdemo/css/bootstrap.css'
    'bundles/ezplatformuiassets/vendors/flag-icon-css/css/flag-icon.min.css'
%}
```
Then we have one asset and working flags.
2. Copy Bootstrap core into DemoBundle
Then we can set output for **cssrewrite** and try something [like this](https://coderwall.com/p/cx1ztw/bootstrap-3-in-symfony2-with-no-extra-bundles)